controllers/modify_backup.go

```diff
@@ -41,6 +41,10 @@ func (s modifyBackup) reconcile(
 		return nil
 	}
 
+	if backup.Spec.EncryptionKeyPath != "" {
```

Comment:
Could we add a comment above this statement why we are skipping the modify sub reconciler (and link the issue from the FDB repo).